container-images/mds-identity/index.ts

```diff
@@ -0,0 +1,28 @@
+/*
+    Copyright 2019 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+// Express local
+import { ApiServer } from '@mds-core/mds-api-server'
+import { api } from '@mds-core/mds-identity'
+import { env } from '@container-images/env-inject'
+
+const { npm_package_name, npm_package_version, npm_package_git_commit, PORT = 4008 } = env()
+
+ApiServer(api).listen(PORT, () =>
+  /* eslint-reason avoids import of logger */
+  /* eslint-disable-next-line no-console */
+  console.log(`${npm_package_name} v${npm_package_version} (${npm_package_git_commit}) running on port ${PORT}`)
```

Comment:
Any reason it's `console.log` and not the `mds-logger`?

Reply:
> Any reason it's `console.log` and not the `mds-logger`?

Historical reasons only. Before Kubernetes, this code only ran when debugging locally. Now that it is running in the pod, it might make sense to convert it (and other APIs as well) to use `mds-logger`.  Or move the startup log to `mds-api-server` since the `ApiServer` call is common to all APIs.

Reply:
Created a tech debt task to update all to using logger:
https://lacunadotai.atlassian.net/browse/LA-10